Framework/src/CcdbDatabase.cxx

```diff
@@ -568,7 +568,10 @@ boost::property_tree::ptree CcdbDatabase::getListingAsPtree(const std::string& p
   std::stringstream listingAsStringStream{ getListingAsString(pathWithMetadata.str(), "application/json", latestOnly) };
 
   boost::property_tree::ptree listingAsTree;
-  boost::property_tree::read_json(listingAsStringStream, listingAsTree);
+  try {
+    boost::property_tree::read_json(listingAsStringStream, listingAsTree);
+  } catch (const boost::property_tree::json_parser::json_parser_error&) {
```

Comment:
Could you add an error with support level, indicating that the listing did not have correct JSON format due to possibly malformed or missing reply from QCDB? The same comment for the other `catch` in the PR.